project/Dependencies.scala

```diff
@@ -42,6 +42,7 @@ object Dependencies {
   val jodaConvert = "org.joda" % "joda-convert" % "1.2"
   val jSoup = "org.jsoup" % "jsoup" % "1.8.3"
   val liftJson = "net.liftweb" %% "lift-json" % "2.6-RC2"
+  val macwire = "com.softwaremill.macwire" %% "macros" % "2.2.3" % "provided"
```

Comment:
> A good example is AdminHttpErrorHandler or AdminLifecycle, or even the Router
modifying the route file (even the order of lines) can modify the order of the parameters in the constructor, or add new parameters, which will trigger a compilation error (that won't be caught by Intellij) whereas macwire is handling that wiring for you at compile time.

I really don't get this point. It it triggers a `compilation` error you should see it immediately with `sbt`. If you don't see a compilation error with `Intellij` that you see with `sbt` then `Intellij` is broken.